Source/GTMSessionFetcherService.m

```diff
@@ -374,17 +374,19 @@ - (void)startFetcher:(GTMSessionFetcher *)fetcher {
 }
 
 // Internal utility. Returns a fetcher's delegate if it's a dispatcher, or nil if the fetcher
-// is its own delegate and has no dispatcher.
+// is its either own delegate and has no dispatcher or swizzled alien.
 - (GTMSessionFetcherSessionDelegateDispatcher *)delegateDispatcherForFetcher:(GTMSessionFetcher *)fetcher {
   GTMSessionCheckNotSynchronized(self);
 
   NSURLSession *fetcherSession = fetcher.session;
   if (fetcherSession) {
     id<NSURLSessionDelegate> fetcherDelegate = fetcherSession.delegate;
-    BOOL hasDispatcher = (fetcherDelegate != nil && fetcherDelegate != fetcher);
-    if (hasDispatcher) {
+      BOOL maybeHasDispatcher = (fetcherDelegate != nil && fetcherDelegate != fetcher);
```

Comment:
Why the indent change here?  Did you mean for this to be part of some new condition?

Reply:
Ouch. Getting aligned with 2 space idents! Thanks.